ui/app/pages/first-time-flow/first-time-flow.component.js

```diff
@@ -113,6 +114,16 @@ export default class FirstTimeFlow extends PureComponent {
               />
             )}
           />
+          <Route
+            path={INITIALIZE_BACKUP_SEED_PHRASE_ROUTE}
```

Comment:
Isn't this unreachable? I would expect that `INITIALIZE_SEED_PHRASE_ROUTE` would always match, because `/initialize/seed-phrase-backup` is a prefix of `/initialize/seed-phrase`, and `exact` isn't set on the above route.

Reply:
No wait, maybe it's the other way around - the other `INITIALIZE_BACKUP_SEED_PHRASE_ROUTE` route is unreachable. React router won't match `/initialize/seed-phrase-backup` as `/initialize/seed-phrase`.